MAUI/NumericEntry/UpDown-Button.md

```diff
@@ -68,3 +68,33 @@ sfNumericEntry.UpDownPlacementMode = NumericEntryUpDownPlacementMode.Inline;
 {% endtabs %}
 
 ![UpDown Placement in .NET MAUI NumericEntry](UpDownButton_images/UpDownButtonPlacement.gif)
+
+## Auto reverse in SfNumericEntry
+
+Auto reverse in numeric up-down buttons allows the control to automatically switch direction when reaching its `minimum` or `maximum` value. When incrementing, it starts at the `minimum` and progresses to the `maximum`, and vice versa.
+
+N> The default value of this property is `false`.
```

Comment:
N> The default value of this property is `false.`

Reply:
@Chrispineagunja,
I don't see any changes in this line.

Reply:
I've seen your changes please ignore it.

Reply:
Changed the content as mentioned.